.github/workflows/diff_report.yml

```diff
@@ -85,6 +85,8 @@ jobs:
           sed 's/Report label: //' temp > report_label
           echo "report_label=$(cat report_label)" >> "$GITHUB_OUTPUT"
 
+          echo "$GITHUB_OUTPUT"
+
```

Comment:
Let put some text upfront to explain output

Reply:
Done.

Reply:
it is kind of weird, that GITHUB_OUTPUT is not self printed.
it is desinged to be self printed.
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

is this PR is experiment ? to "hack" Gitub and print it.
I thought that github redirect all output to his internals, that is why they need this variable.

Reply:
I am not sure what "self-printed" means, but `output` in this context is the output of a given step, to be passed to the next one.

Reply:
>is this PR is experiment ? to "hack" Gitub and print it.

Sort of. It is a good sanity check to know what we are passing to the next step. You cannot ssh into containers like you can with CircleCI for debugging, so it will also be helpful for that.